app/controllers/books_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to book_url(@book), notice: t('.notice') }
+        format.html { redirect_to book_url(@book), notice: t('common.create_notice', name: Book.model_name.human) }
```

Comment:
```suggestion
        format.html { redirect_to book_url(@book), notice: t('controllers.common.create_notice', name: Book.model_name.human) }
```

コントローラ用の翻訳であることがわかるようなキー名の方がいいかも（ルールが決まっているわけではないですが）

Reply:
controllersの翻訳の参照を修正しました。
そのため、翻訳ファイルymlファイルを一つにまとめ、controllersキーの翻訳を移動いたしました。